src/RadialGauge/RadialGauge.story.tsx

```diff
@@ -113,7 +112,7 @@ export const Stacked = () => {
       height={300}
       width={700}
       minValue={0}
```

Comment:
If we do this i think we should do the same for min value no?

Reply:
> If we do this i think we should do the same for min value no?

Idk. I think min in most cases 0

Reply:
Let's go ahead for consistency please.